model/textparse/openmetricsparse.go

```diff
@@ -467,6 +585,22 @@ func (p *OpenMetricsParser) parseLVals(offsets []int, isExemplar bool) ([]int, e
 	}
 }
 
+// isCreatedSeries returns true if the current series is a _created series.
+func (p *OpenMetricsParser) isCreatedSeries() bool {
+	var newLbs labels.Labels
+	p.Metric(&newLbs)
+	name := newLbs.Get(model.MetricNameLabel)
+	switch p.mtype {
+	case model.MetricTypeCounter, model.MetricTypeSummary, model.MetricTypeHistogram:
+		if strings.HasSuffix(name, "_created") {
+			return true
+		}
+	default:
+		break
+	}
+	return false
+}
+
 // parseMetricSuffix parses the end of the line after the metric name and
 // labels. It starts parsing with the provided token.
 func (p *OpenMetricsParser) parseMetricSuffix(t token) (Entry, error) {
```

Comment:
Is there ANY case where err == nil but Entry is NOT EntrySeries? I don't think so. Then we can reduce this method to only return error and assume `EntrySeries` after calling this method with success, no?